tox.ini

```diff
@@ -4,5 +4,5 @@ envlist = py27, py34, py35, pypy, pypy3
 [testenv]
 deps = -r{toxinidir}/tests/requirements.txt
 commands =
-    flake8
+    flake8 --ignore=E741,W605
```

Comment:
I would prefer a patch that fixes #23 over this masking of the warnings (as pedantic and annoying as they are).